File: php/testcode/benchmark_test_00461.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00461(BenchmarkRequest $req): BenchmarkResponse {
    $input = $req->param('q');

    $data = ['query' => $input, 'results' => []];

    return BenchmarkResponse::json($data);
}

File: rust/testcode/benchmark_test_00461.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let id = req.param("id");

    if id.is_empty() {
        return super::shared::BenchmarkResponse::bad_request(
            r#"{"error":"missing_id","status":400}"#,
        );
    }

    let body = r#"{"error":"not_found","status":404}"#;

    super::shared::BenchmarkResponse { status: 404, body: body.to_string() }
}
